lib/rules/no-extra-parens.js

```diff
@@ -256,6 +256,23 @@ module.exports = {
         }
 
         /**
+         * Determines whether a node should be followed by an additional space when removing parens
+         * @param {ASTNode} node node to evaluate; must be surrounded by parentheses
+         * @returns {boolean} `true` if a space should be inserted after the node
+         * @private
+         */
+        function requiresAfterSpace(node) {
```

Comment:
Nitpick: `requiresTrailingSpace` might be a little clearer for this function name